functions/Restore-DbaDatabase.ps1

```diff
@@ -533,7 +533,7 @@ function Restore-DbaDatabase {
             if ($DestinationLogDirectory -ne '') {
                 if ((Test-DbaSqlPath -Path $DestinationLogDirectory -SqlInstance $SqlInstance -SqlCredential $SqlCredential) -ne $true) {
                     if ((New-DbaSqlDirectory -Path $DestinationLogDirectory -SqlInstance $SqlInstance -SqlCredential $SqlCredential).Created -ne $true) {
-                        Write-Warning "$FunctionName - DestinationLogDirectory $DestinationLogDirectory does not exist, and could not be created on $SqlInstance"
+                        Write-Message -Level Warning -Message "$FunctionName - DestinationLogDirectory $DestinationLogDirectory does not exist, and could not be created on $SqlInstance" 
                         break
```

Comment:
Use Stop-Function instead. Including the function name in messages is no longer needed.